### DISCUSSION TITLE
Github campus Ambassador program

### DISCUSSION THREAD

**Discussion by ompatil19** [community/community#50392] [GitHub Education]
I applied for the program in the February batch and the results were supposed to be announced by now as the email said 2 weeks but I haven't received any email regarding the same application being accepted/rejected. 
Can someone please help me out

**Comment by Ntshangase** [comment]
Hello @ompatil19 
The response emails have been sent out but if you haven't received yours, You can check your status here : https://education.github.com/campus_experts/status
